cli/src/aws_sso_util/populate_profiles.py

```diff
@@ -244,6 +265,10 @@ def populate_profiles(
         profile_name_formatter = get_formatter(profile_name_include_region, region_format, no_region_format)
         if profile_name_trim_account_name_patterns or profile_name_trim_role_name_patterns:
             profile_name_formatter = get_trim_formatter(profile_name_trim_account_name_patterns, profile_name_trim_role_name_patterns, profile_name_formatter)
+        if profile_name_account_name_case_transform:
```

Comment:
Let's make this work like `get_trim_formatter`, where it operates on both fields at once, rather than having the general-purpose field transform as it is. make it `get_case_formatter(profile_name_account_name_case_transform, profile_name_role_name_case_transform, profile_name_formatter)`

Reply:
updated